--- a/Content/Algorithms/Brute Force/Selection Sort.html
+++ b/Content/Algorithms/Brute Force/Selection Sort.html
@@ -74,6 +74,7 @@
     <li>Repeat until the unsorted portion contains only one element.</li>
   </ol>
 
+<section id="demo" section-title="Interactive Demo">
 <div class="embeddedDemoContainer">
   <iframe class="embeddedDemo"
           src="/Algorithms/Content/Demos/Brute Force/Selection Sort Demo.html"
@@ -81,6 +82,7 @@
           name="selection-sort-demo">
   </iframe>
 </div>
+</section>
 
 
 
